style/layout.css

```diff
@@ -0,0 +1,88 @@
+header #a {
+  width: 480px;
+  height: 75px;
+  background-color: #5f9ea0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#b {
+  width: 480px;
+  height: 150px;
+  background-color: #e0ffff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#c {
+  width: 480px;
+  height: 125px;
+  background-color: #00ffff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#d {
+  width: 480px;
+  height: 125px;
+  background-color: #40e0d0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#section1 {
+  display: flex;
+}
+
+#section1 #e {
+  width: 240px;

```

Comment:
same with these two, you only need to say `#e` and `#f` and not have `#section1`

Reply:
fixed!